src/tasks/tasks.validators.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNotEmpty, IsNumberString } from 'class-validator';
+
+export type Task = {
+    id: number;
+    status: string;
+    taskName: string;
+    startDate: Date;
+    finishDate: Date;
+  };
+
+
+export class ValidateId {
+  @IsNotEmpty()
+  @IsNumberString()
+  id: number;
+}
+
+export class RequestIsNotEmpty {
+    @IsNotEmpty() id: number;
+    @IsNotEmpty() status: string;
```

Comment:
I added validation for checking is this request is not empty (by checking every field)? Is there more proficient way to do so?

Also does I need to do more validation check when working with DB?